apis/v1/gateway_types.go

```diff
@@ -739,9 +739,10 @@ type GatewayAddress struct {
 	//
 	// Examples: `1.2.3.4`, `128::1`, `my-ip-address`.
 	//
+	// +optional
 	// +kubebuilder:validation:MinLength=1
```

Comment:
Since we decided against making this a pointer, we need to remove the minLength to make this optional
```suggestion
```